README.MD

```diff
@@ -175,7 +175,21 @@ and paste in the following contents.
   "horizon_middleware_protocol": "https",
   "horizon_middleware_host": <horizon_middleware_host>,
   "horizon_middleware_username": <your_horizon_middleware_username>,
-  "horizon_middleware_password": <your_horizon_middleware_password>
+  "horizon_middleware_password": <your_horizon_middleware_password>,
+  "dl": {
```

Comment:
I like the hierarchical naming here, that will easily lend itself for automation.